Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java

```diff
@@ -0,0 +1,100 @@
+package io.branch.referral;
```

Comment:
Pretty sure this file was removed cc @JagadeeshKaricherla-branch

Reply:
Yes here https://github.com/BranchMetrics/android-branch-deep-linking-attribution/pull/1133/files

Reply:
ok makes sense since this is of non-master branch.